### PR TITLE
[EI-223] geolocation updates

### PR DESCRIPTION
**Pull request by jazzsequence**
This PR contains is quality-of-life updates as well as specific updates to the code to make the geolocation shortcode function compatible with 0.3.x of the plugin.

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -253,14 +253,16 @@ function do_shortcode_content( $atts, string $content = '' ) : string {
...
+	if ( empty( $geos ) || empty( $atts ) ) {
 		return '';
 	}
 
 	foreach ( $atts as $label => $value ) {
 		// Set up initial negation parameters.
 		$negate = 0;
 		$inline_negate = 0;
+		$label = $label === 'country' ? 'country-code' : $label;
+		$label = $label === 'continent' ? 'continent-code' : $label;
```
Comment (CdrMarks) [inline inc/namespace.php:265]: Are lines 264-265 temporary while the old variable names are phased out?

File: tests/shortcodesTest.php [hunk 2]
```diff
@@ -428,8 +383,8 @@ public function testDoShortcodeContent() {
 			$this->fail( 'Failing this test because the country is not US or the region is not UT.' );
 		}
 		$this->assertEquals(
-			Shortcodes\do_shortcode_content( $shortcode, $content ),
 			'',
+			Shortcodes\do_shortcode_content( $shortcode, $content ),
```
Comment (CdrMarks) [inline tests/shortcodesTest.php:387]: I see several diffs where the order of the function calls are changed.  Can you explain this a bit for me?

**Review by CdrMarks — APPROVED** [review]
I've some questions, but these are not blocking.

File: inc/namespace.php (hunk 1 above)
Comment (jazzsequence) [reply inc/namespace.php:265]: Not necessarily. Part of the requirement for holding onto `continent` and `country` as nouns that are supported is the shortcode itself. I don't want people to need to use [geoip country-*code*] or [geoip continent-*code*] -- I would prefer they use the expected `[geoip country]` which is easier to remember and use. To support that with the updated header names, we need to check for both things. This is especially important when we're dealing with the negations where the shortcode attribute is something like `not_country=us` -- I don't want to force people to use `not_country-code=us` which is less intuitive.

File: tests/shortcodesTest.php (hunk 2 above)
Comment (jazzsequence) [reply tests/shortcodesTest.php:387]: Yeah, I found through testing this that the _expected_ result should be the first parameter when doing `assertEquals` checks. This became evident through messages like this:

```
[geoip-content continent="NA" not_country="US"] does not match expected output.
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-''
+'This should be hidden from US visitors.'
```

In the case above the _Expected_ is being shown as the _Actual_ result which is inaccurate.

**Pull request merged**